src/sentry/db/models/base.py

```diff
@@ -74,7 +74,11 @@ def _update_tracked_data(self):
             data = {}
             for f in self._meta.fields:
                 try:
-                    data[f.column] = self.__get_field_value(f)
+                    if isinstance(self.__get_field_value(f), (int, float, complex,
```

Comment:
Let's only execute `self.__get_field_value(f)` once. Now we're doing it twice. Once for the `isinstance` check, then again storing it.

This also misses `long` which is a pretty common value for us, so you can summarize with `six.integer_types` which covers `(int, long)`

What about unicodes? They are immutable too.

Why don't we flip this to explicitly handle bitfield data for now? Seems easier to just explicitly handle the edge cases than to keep track of what types are immutable and which aren't. At least for now.